lib/internal/vm/module.js

```diff
@@ -338,19 +331,13 @@ class SourceTextModule extends Module {
   }
 
   get dependencySpecifiers() {
-    if (this[kWrap] === undefined) {
-      throw new ERR_VM_MODULE_NOT_MODULE();
-    }
-    if (this[kDependencySpecifiers] === undefined) {
-      this[kDependencySpecifiers] = this[kWrap].getStaticDependencySpecifiers();
-    }
+    validateInternalField(this, kDependencySpecifiers, 'SourceTextModule');
+    this[kDependencySpecifiers] ??= ObjectFreeze(this[kWrap].getStaticDependencySpecifiers());
```

Comment:
Isn't the freezing (from https://github.com/nodejs/node/pull/49720) semver-major?

Reply:
Looks like it, I guess we cannot backport this then